base_layer/transactions/src/consensus.rs

```diff
@@ -30,6 +37,9 @@ pub struct ConsensusRules {
     emission_schedule: EmissionSchedule,
     /// Current version of the blockchain
     blockchain_version: u16,
+    /// The FTL of the blockchain in seconds
```

Comment:
Please explain what FTL is

Reply:
Upcoming RFC explains what it is, FTL is widely used term in timestamps of blockchains. But I expanded the comment